app/assets/stylesheets/config/_fonts.css

```diff
@@ -0,0 +1,59 @@
+html {
+  font-family: proxima-nova, sans-serif;
+  font-weight: 600;
+}
+
+h1 {
+  font-family: eds-market-bold-slant, sans-serif;
+  font-size: var(--font-size-xl);
+}
+
+h2, h3, h4 {
+  font-weight: 900;
+  text-transform: uppercase;
+}
+
+h5, h6 {
+  font-weight: 600;
+  text-transform: uppercase;
+}
+
+p {
+  font-weight: 600;
+  font-size: var(--font-size-s);
+  line-height: 1.5;
+}
+
+h2 {
+  font-size: var(--font-size-l);
+}
+
+h3 {
+  font-size: var(--font-size-m);
+}
+
+h4, h5 {
+  font-size: var(--font-size-s);
+}
+
+h6 {
+  font-size: var(--font-size-xs);
+}
```

Comment:
**suggestion:** should we move these to `base`?

Reply:
I thought of that, but maybe the base would get too bloated? I'm not sure, WDYT?

Reply:
I think it depends on what makes more sense for your future self. Bloat or not is rarely the right question for me. In these cases, I tend to think in terms of what is easier for me to navigate. If you feel like fonts is better, then let's go for fonts. For me personally, if there is not a tag, I search in base haha

Reply:
I think for me keeping it in a fonts file might be easier. I'll leave it like this for now and see how it works 🙂